blueprints/component/index.js

```diff
@@ -149,37 +134,7 @@ module.exports = {
   fileMapTokens(options) {
     let commandOptions = this.options;
 
-    if (commandOptions.pod) {
-      return {
-        __path__() {
-          return path.join(options.podPath, options.locals.path, options.dasherizedModuleName);
-        },
-        __templatepath__() {
-          return path.join(options.podPath, options.locals.path, options.dasherizedModuleName);
-        },
-        __templatename__() {
-          return 'template';
-        },
-      };
-    } else if (
-      !this.EMBER_GLIMMER_SET_COMPONENT_TEMPLATE ||
-      commandOptions.componentStructure === 'classic'
-    ) {
-      return {
-        __path__() {
-          return 'components';
-        },
-        __templatepath__() {
-          return 'templates/components';
-        },
-        __templatename__() {
-          return options.dasherizedModuleName;
-        },
-      };
-    } else if (
-      this.EMBER_GLIMMER_SET_COMPONENT_TEMPLATE &&
-      commandOptions.componentStructure === 'flat'
-    ) {
+    if (this.EMBER_GLIMMER_SET_COMPONENT_TEMPLATE && commandOptions.componentStructure === 'flat') {
```

Comment:
I think the `EMBER_GLIMMER_SET_COMPONENT_TEMPLATE` code can also be removed but that's unrelated to this PR.